templates/gpg/gpg-batch-file.j2

```diff
@@ -0,0 +1,13 @@
+%echo Generating a basic OpenPGP key
+Key-Type: RSA
+Key-Length: 4096
+Subkey-Type: ELG-E
+Subkey-Length: 4096
+Name-Real: {{ archivematica_src_configure_gpg.key_user }}
+Name-Comment: {{ archivematica_src_configure_gpg.key_comment }}
+Name-Email: {{ archivematica_src_configure_gpg.key_mail }}
+Expire-Date: 0
+Passphrase: passphrase
```

Comment:
Can we use a configuration parameter for this, and if it's not available just use a blank passphrase?

Reply:
We are using empty string when creating the keys from GUI:

https://github.com/artefactual/archivematica-storage-service/blob/stable/0.14.x/storage_service/common/gpgutils.py#L31

We can add a new field to the dict for the passphrase (it's free), good idea. Thanks!